packages/devtools_app/lib/src/shared/analytics/constants.dart

```diff
@@ -22,6 +22,7 @@ final debugger = ScreenMetaData.debugger.id;
 final logging = ScreenMetaData.logging.id;
 final appSize = ScreenMetaData.appSize.id;
 final vmTools = ScreenMetaData.vmTools.id;
+const console = 'console';
```

Comment:
this doesn't appear to be used anywhere. Let's remove

Reply:
Used for help dialog.